test/blackbox-tests/test-cases/lib-errors/dune

```diff
@@ -0,0 +1,28 @@
+(library
+ (name a)
+ (libraries b)
+ (modules))
+
+(library
+ (name b)
+ (libraries c)
+ (modules))
+
+(library
+ (name c)
+ (libraries a)
+ (modules))
+
+(executable
+ (name cycle)
+ (libraries a)
+ (modules cycle))
+
+(rule (with-stdout-to cycle.ml (echo "")))
+
+(executable
+ (name select_error)
+ (libraries (select x from))
```

Comment:
Maybe this form should just be an error?

Reply:
Possibly yh